docs/source/examples/substrafl/get_started/run_mnist_torch.ipynb

```diff
@@ -249,16 +249,16 @@
         "import numpy as np\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        y_pred = predictions
</pre><p>I guess we can use <code>predictions</code> directly, no? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Substra/substra-documentation/pull/376/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>